packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -1,25 +1,36 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import React from 'react';
-import { participantListStack, participantListStyle, participantListWrapper, listHeader } from './styles/Chat.styles';
-import { OnRenderAvatarCallback, ParticipantList, ParticipantMenuItemsCallback } from '@internal/react-components';
+import {
+  participantListStack,
+  participantListStyle,
+  participantListWrapper,
+  listHeader
+} from '../ChatComposite/styles/Chat.styles';
```

Comment:
These styles should also move to `common`. Stuff from `common` shouldn't import from `ChatComposite`.